src/ray/stats/metric_defs.h

```diff
@@ -12,16 +12,53 @@
 /// You can follow these examples to define your metrics.
 
 static Gauge CurrentWorker("current_worker",
-                           "This metric is used for report states of workers. "
+                           "This metric is used for reporting states of workers."
                            "Through this, we can see the worker's state on dashboard.",
-                           "1 pcs", {NodeAddressKey, LanguageKey, WorkerPidKey});
+                           "1 pcs", {LanguageKey, WorkerPidKey});
+
+static Gauge CurrentDriver("current_driver",
+                           "This metric is used for report states of drivers.", "1 pcs",
+                           {LanguageKey, DriverPidKey});
 
 static Count TaskCountReceived("task_count_received",
-                               "The count that the raylet received.", "pcs",
-                               {NodeAddressKey});
+                               "Number of tasks received by raylet.", "pcs", {});
 
 static Histogram RedisLatency("redis_latency", "The latency of a Redis operation.", "us",
                               {100, 200, 300, 400, 500, 600, 700, 800, 900, 1000},
-                              {NodeAddressKey, CustomKey});
+                              {CustomKey});
+
+static Gauge LocalAvailableResource("local_available_resource",
+                                    "The available resource of this node.", "pcs",
+                                    {ResourceNameKey});
+
+static Gauge LocalTotalResource("local_total_resource",
+                                "The total resource of this node.", "pcs",
+                                {ResourceNameKey});
+
+static Gauge ActorStats("actor_stats", "Stat the metric values of actor in raylet.",
```

Comment:
```suggestion
static Gauge ActorStats("actor_stats", "Stat metrics of the actors in raylet.",
```
Also apply to the following fields